backend/graminsta/post/views.py

```diff
@@ -1,3 +1,42 @@
-# from django.shortcuts import render
+"""
+Views for post and comment APIs
+"""
 
-# Create your views here.
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.parsers import MultiPartParser
+from .serializers import PostSerializer
+from .services import create_post
+
+
+class PostRecordView(APIView):
+    """
+    A class based view for creating Post Record
+    """
+    parser_classes = [MultiPartParser]
+
+    def post(self, request):
+        """
+        Create a Post record
+
+        Parameters
+        ----------
+        request: json format
+            Data containing publisher_id, description and image binary data
+
+        Returns
+        ----------
+        response: json format
+            Newly created post
+        """
+
+        # _ is not allowed in header key
+        publisher_id = int(request.META.get("HTTP_PUBLISHERID"))
```

Comment:
I think to get current user, you can use `request.user` (if authentication works, otherwise you can leave a TODO here)`

Reply:
Add TODO here